charts/zeebe-benchmark/templates/starter.yaml

```diff
@@ -47,6 +51,28 @@ spec:
             - name: LOG_LEVEL
               value: {{ .Values.starter.logLevel | quote }}
             {{- end }}
+            {{- if $.Values.saas.enabled }}
+            - name: ZEEBE_ADDRESS
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "zeebe-benchmark.credentials-name" $ }}
```

Comment:
:question: Is the address really a secret? :shrug:

Reply:
I guess it doesn't matter